src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -138,6 +138,21 @@ async fn verify_standard_transaction(
     let recipient = parse_recipient(&request.recipient)?;
 
     if !request.data.is_empty() {
+        confirm::confirm(&confirm::Params {
+            title: "Unknown\ncontract",
+            body: "You will be shown\nthe raw data field\n of the transactoin.",
```

Comment:
"... transact**oi**n" typo.

also there's a space in `\n of the transaction`. wouldn't that display the text misaligned like so:

```
You will be shown
the raw data field
 of the transaction.
```

or make it shorter? not sure anyone cares whether it's a *field*.

```
You will be shown
the raw transaction data.
```

Reply:
Fixed space.

if you want (and you should) verify it, you need to know that it is the data field and not the full raw tx.

Reply:
> it is the data field and not the full raw tx

and yet the 3rd confirmation screen title says "Transaction data".

Reply:
:) 

not sure you still want text split in 3 lines. seems like it might fit just on two? but not sure.

Reply:
does not fit unfortunately